pages/sdk/protocol-kit/guides/safe-deployment/address-replication.mdx

```diff
@@ -0,0 +1,173 @@
+import { Steps } from "nextra/components";
+
+# Safe Address Replication Across Chains
+
+In this guide, you will learn how to replicate a Safe address across different chains using the Protocol Kit. This process includes initializing the Protocol Kit, configuring the Safe, predicting the Safe address on different chains, and executing the deployment.
+
+We will demonstrate how to deploy a Safe account on both the Sepolia and Chiado chains with the same address using the Protocol Kit.
+
+For more detailed information, see the [Protocol Kit Reference](../../../protocol-kit/reference/safe.mdx).
+
+## Prerequisites
+
+- [Node.js and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+
+## Install dependencies
+
+First, you need to install the Protocol Kit.
+
+```bash
+pnpm add @safe-global/protocol-kit
+```
+
+## Steps
+
+<Steps>
+
+### Imports
+
+Begin by importing the `Safe` class from the Protocol Kit and chain configuration from viem.
+
+{/* <!-- vale off --> */}
+
+```typescript
+import Safe, { SafeDeploymentConfig, SafeAccountConfig } from '@safe-global/protocol-kit'
+import { waitForTransactionReceipt } from 'viem/actions'
+import { sepolia, gnosisChiado } from 'viem/chains'
+```
+
+{/* <!-- vale on --> */}
+
+### Create a signer
+
+Firstly, you need to get a signer to execute the deployment transaction in each chain. this example uses a private key, but any way to get an `EIP-1193` compatible signer can be used.
+
+{/* <!-- vale off --> */}
+
+```typescript
+const SIGNER_PRIVATE_KEY = // ...
+```
+
+{/* <!-- vale on --> */}
+
+### Define the Safe Configuration
+
+{/* <!-- vale off --> */}
+
+Define the configuration for your Safe. This includes the `owners`, `threshold`, deployment type, salt nonce for address predictability, and the version of the Safe.
```

Comment:
Link to the protocol kit reference to get more info about the options

Reply:
Some properties are also mentioned as `code` and some others with words. Either one way or the order.